docs/manual/manual.tex

```diff
@@ -41,7 +41,8 @@
 \input{map-key-checker.tex}
 \input{interning-checker.tex}
 \input{lock-checker.tex}
-%\input{index-checker.tex}
+% Include index-checker.tex EXCEPT on buffalo.cs.washington.edu
+\IfFileExists{/scratch/secs-jenkins/java/jdk1.7.0}{\relax}{\input{index-checker.tex}}
```

Comment:
Yes, I made that change.  It works as desired for the PDF version of the manual, but not for the HTML version.  I'll commit a version that works for both PDF and HTML separately.